DocIndexRetriever/docker/Dockerfile

```diff
@@ -14,6 +12,11 @@ RUN useradd -m -s /bin/bash user && \
     mkdir -p /home/user && \
     chown -R user /home/user/
 
+
+RUN cd /home/user/ && \
+    git clone https://github.com/opea-project/GenAIComps.git
+
```

Comment:
LGTM

Reply:
I don't suggest to do this, we usually need to freeze the GenAIComps branch in the image build, like v1.0rc, so better to keep use COPY.

Reply:
> I don't suggest to do this, we usually need to freeze the GenAIComps branch in the image build, like v1.0rc, so better to keep use COPY.

Done